files/js-cookie/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "js-cookie",
+  "repo": "js-cookie/js-cookie",
+  "files": {
+    "include": ["src/js.cookie.js"]
```

Comment:
What if we do not provide a minified file in the source repository? See [src/js.cookie.js](https://github.com/js-cookie/js-cookie/blob/master/src/js.cookie.js), the minified file is generated at build time.

The docs says:

> array that specifies files and dirs that will be copied from **project package**.

In this case the **project package** is the github, right?